lib/renderer/api/screen.js

```diff
@@ -1,3 +1,9 @@
 'use strict'
 
-module.exports = require('electron').remote.screen
+const { remote } = require('electron')
+
+if (remote) {
+  module.exports = remote.screen
+} else {
+  throw new Error('screen requires remote, which is not enabled')
```

Comment:
This logic is duplicated in a few places, can we just have a helper

```js
requireRemoteOrThrow()
```

or something like that

Reply:
done